src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -488,16 +488,18 @@ Tests:
     - a: "64.83.195.254"
     - a: "12.171.202.78"
 
-  # The commented portions of this test should work, but do not. This appears to be due to a bson or
-  # testlib error in formatting of bson. One case will have an extra whitespace than the other.
   - Name: Actor ID
     GivenTemplate:
-      # int: {^ActorId: {}}
       string: {^ActorIdString: {}}
     ThenReturns:
-    #    - { "int" : { "$numberLong" : "2000000000"}, "string" : "2000000000" }
     - "string": "2"
 
+  - Name: Actor ID
+    GivenTemplate:
+      int: {^ActorId: {}}
+    ThenReturns:
+    - { "int" : { "$numberLong" : "2000000000"}, "string" : "2000000000" }
```

Comment:
I don't think this works - there's no `string` key in the `GivenTemplate`

Reply:
I think the test case got split into two distinct test cases here, when the original had string and int.